<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Jumei Test"
           tests="3"
           failures="1"
           errors="0"
           skipped="0"
           time="24.655">
  <properties />
            <testcase name="settle" time="2.848" classname="com.jumei.autotest.testcase.NormalShoppingTest">
                <failure type="java.lang.AssertionError"
                  message="expected [true] but found [false]"
              >
<![CDATA[java.lang.AssertionError: expected [true] but found [false]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:494)
  org.testng.Assert.assertTrue(Assert.java:42)
  org.testng.Assert.assertTrue(Assert.java:52)
  com.jumei.autotest.testcase.NormalShoppingTest.settle(NormalShoppingTest.java:167)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:606)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
  org.testng.SuiteRunner.access$000(SuiteRunner.java:38)
  org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:382)
  org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
  java.util.concurrent.FutureTask.run(FutureTask.java:262)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
  java.lang.Thread.run(Thread.java:745)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="bugGoods" time="17.655" classname="com.jumei.autotest.testcase.NormalShoppingTest" />
                <testcase name="login" time="4.152" classname="com.jumei.autotest.testcase.NormalShoppingTest" />
      
</testsuite>
